core/src/main/java/org/mule/processor/AsyncInterceptingMessageProcessor.java

```diff
@@ -64,7 +65,13 @@ public AsyncInterceptingMessageProcessor(ThreadingProfile threadingProfile,
     {
         this.doThreading = threadingProfile.isDoThreading();
         this.threadTimeout = threadingProfile.getThreadWaitTimeout();
-        workManager = threadingProfile.createWorkManager(name, shutdownTimeout);
+
+        // (Add one to maximum number of active threads to account for the service
+        // work item that is running continuously and polling the SEDA queue.)
+        ThreadingProfile dedicatedPollingThreadThreadingProfile = new ChainedThreadingProfile(threadingProfile);
+        dedicatedPollingThreadThreadingProfile.setMuleContext(threadingProfile.getMuleContext());
+        dedicatedPollingThreadThreadingProfile.setMaxThreadsActive(threadingProfile.getMaxThreadsActive() + 1);
```

Comment:
what if instead of always adding `1`, only if the configured value was `1` then it is changed to `2`? Would that work fir the test case you just added?